### PR TITLE
docs(ReadMe): Updated readme. Minor changes on smart contract and deployment script

### PR DESCRIPTION
**Pull request by Abhishekkochar**
#### What this PR does / why we need it:
Updated readme as per the new contract.
Change visibility of a fn() in bridge smart contract
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: packages/smartcontracts/src/contracts/BridgeV1.sol [hunk 1]
```diff
@@ -252,14 +252,14 @@ contract BridgeV1 is UUPSUpgradeable, EIP712Upgradeable, AccessControlUpgradeabl
     /**
      * @notice Used by addresses with Admin and Operational roles to add a new supported token and daily allowance
      * @param _tokenAddress The token address to be added to supported list
-     * @param _currentCap maximum balance of tokens the contract can hold per `_tokenAddress`
+     * @param _tokenCap maximum balance of tokens the contract can hold per `_tokenAddress`
```
Comment (Abhishekkochar) [inline packages/smartcontracts/src/contracts/BridgeV1.sol:255]: Only changing the parameter name to provide a better description.

File: packages/smartcontracts/src/contracts/BridgeV1.sol [hunk 2]
```diff
@@ -368,7 +368,7 @@ contract BridgeV1 is UUPSUpgradeable, EIP712Upgradeable, AccessControlUpgradeabl
     /**
      * @notice to check whether a token is supported
      */
-    function isSupported(address _tokenAddress) public view returns (bool) {
+    function isSupported(address _tokenAddress) external view returns (bool) {
```
Comment (Abhishekkochar) [inline packages/smartcontracts/src/contracts/BridgeV1.sol:371]: Change the visibility to external. No need for this to be public.

File: README.md [hunk 3]
```diff
@@ -45,37 +45,46 @@ Once approved, user will call the `bridgeToDeFiChain()` function with following
...
 ### Remove supported token
 
-Only addresses with the Admin and Operational role can call the `removeSupportedTokens()` function.
+Only addresses with the Admin and Operational role can call the `removeSupportedTokens()` function. This also sets the `_tokenCap` to `0`.
 
-### Withdraw ERC20
+### Withdraw
 
-Only the Admin can call the `withdraw()` function with the token's address and amount.
+`withdraw()` function when called will withdraw an ERC20 token. Only the address with the Admin role can call this function.
```
Comment (weiyuan95) [inline README.md:58]: For ETH as well right?

File: README.md [hunk 4]
```diff
@@ -45,37 +45,46 @@ Once approved, user will call the `bridgeToDeFiChain()` function with following
...
 
-### Change Daily Allowance
+### FlushFund
 
-Both the Admin and Operational addresses can change the `_dailyAllowance` (the new daily allowance) and `_newResetTimeStamp` (the timestamp when the token will start being supported) via the `changeDailyAllowance()` function. The changes will only come into effect when the current timestamp has reached the `_newResetTimeStamp`. This new timestamp will have to be at least 24 hours in the future, if not the function will revert with `INVALID_RESET_EPOCH_TIME`
+`flushFund` function to flush the excess funds `(token.balanceOf(Bridge) - tokenCap)` across supported tokens to a hardcoded address (`flushReceiveAddress`) anyone can call this function.
```
Comment (weiyuan95) [inline README.md:62]: Applies to ETH as well.

File: README.md [hunk 5]
```diff
@@ -45,37 +45,46 @@ Once approved, user will call the `bridgeToDeFiChain()` function with following
 
 ### Add supported token
 
-Only addresses with the Admin and Operational roles can call the `addSupportedTokens()` function. This sets the `_dailyAllowance` for an ERC20 token identified by its `_tokenAddress`. The `_startAllowanceTimeFrom` also represents when this token 'goes live'
-User are not allowed to bridge more than the dailyAllowance per day.
+Only addresses with the Admin and Operational roles can call the `addSupportedTokens()` function. This sets the `_tokenCap` for an ERC20 token identified by its `_tokenAddress`. All added tokens will be instantly supported by the bridge.
```
Comment (weiyuan95) [inline README.md:48]: Applies to ETH as well.

File: README.md (hunk 3 above)
Comment (Abhishekkochar) [reply README.md:58]: Will update it according to eth as well

File: README.md [hunk 6]
```diff
@@ -45,37 +45,48 @@ Once approved, user will call the `bridgeToDeFiChain()` function with following
...
 ### Change relayer address
 
 Both the Admin and Operational addresses can change `relayerAddress`.
+
 The relayer address will primarily be used for verifying the signature that is signed by the server. The server will need to sign with the corresponding private key of the relayer address.
 
 ### Transaction fee change
 
-Only address with admin role can change `transactionFee`. Initial fee will be set to 0.3%. This means that if the user bridges `X` tokens, he will only bridge 99.7% of X. The other 0.3% will be taken as fees.
+Only addresses with admin roles can change `transactionFee`.
```
Comment (cuongquangnam) [inline README.md:79]: should be admin + operational

File: README.md [hunk 7]
```diff
@@ -45,37 +45,48 @@ Once approved, user will call the `bridgeToDeFiChain()` function with following
...
+
+Initial fee will be set to 0.1%. This means that if the user bridges `X` tokens, 99.9% of X will be bridged. The other 0.1% will be taken as fees and sent to `communityWallet`.
+
+### Change Tx Fee Address
+
+Only addresses with admin roles can change `communityWallet`. This is where the tx fees upon bridging will go.
```
Comment (cuongquangnam) [inline README.md:85]: admin + operational

File: README.md [hunk 8]
```diff
@@ -45,37 +45,48 @@ Once approved, user will call the `bridgeToDeFiChain()` function with following
...
+
+### Change Token Cap
+
+Only addresses with admin roles can change `tokenCap`.
```
Comment (cuongquangnam) [inline README.md:89]: admin + operational

File: README.md [hunk 9]
```diff
@@ -45,37 +45,48 @@ Once approved, user will call the `bridgeToDeFiChain()` function with following
...
 
 ### Modify admin and operational address
```
Comment (cuongquangnam) [inline README.md:91]: should we add who has the right to grant roles? (let's say for who has the right to grant operational roles)

File: README.md (hunk 6 above)
Comment (Abhishekkochar) [reply README.md:79]: with `admin roles`, I meant both admin and operation addresses. However, I should change it to better describe the addresses.

File: README.md (hunk 9 above)
Comment (Abhishekkochar) [reply README.md:91]: Yes, we should. More the better.

**Review by cuongquangnam — APPROVED** [review]

**Review by weiyuan95 — APPROVED** [review]

**Pull request merged**